client/src/components/ActiveChat/InputBox/InputBox.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, FormControl, FilledInput } from '@material-ui/core';
+import { useConversation } from '../../../context/useActiveConvoContext';
+import { useAuth } from '../../../context/useAuthContext';
+import useStyles from './useStyles';
+import { sendMessage } from '../../../helpers/APICalls/conversation';
+
+const InputBox = () => {
+  const classes = useStyles();
+
+  const { conversations } = useConversation();
+  const { loggedInUser } = useAuth();
+
+  const handleSubmit = async (event: any) => {
+    event.preventDefault();
+
+    const payLoad = {
```

Comment:
Maybe rename this variable to increase semantic meaning (Eg: message/newMessage)